index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="css/style.css">
+    <script src="https://use.fontawesome.com/bba5c6ae87.js"></script>
+    <script src="https://use.fontawesome.com/83ac61cd0f.js"></script>
+    <title>My Portfolio</title>
+</head>
+<body>
+<header class="wrapper">
+    <nav class="flex-nav">
+        <ul>
+            <li ><a class="logo" href="#">Chidiebere</a> </li>
+            <li class="menu-icon toggle-icon"><i class="fa fa-bars" aria-hidden="true"></i></li>
+        </ul>
+    </nav>
+</header>
+
+
+<section>
+    <div class="headline">
+
+        <h1 class="intro name">Hey There. I'm Chidiebere Ojingwa</h1>
+        <p class="intro profession">I'am a Software Developer</p>
```

Comment:
- [ ] You may consider this as your second title. Enclose it within `hx` tags preferably `h2` tags instead of `p` tags